woocommerce/woocommerce-gutenberg-products-block
# Add/GitHub status badges

This PR will add badges for the following GitHub actions at the top of the README:
- PHP Coding standards
- JavaScript and CSS Linting
- PHP, JavaScript, and E2E tests

I don't think the other workflows are very useful to have badges for.


Fixes #3293
Depends on #3544 

### Screenshots


<img width="945" alt="Screenshot 2020-12-22 at 14 22 56" src="https://user-images.githubusercontent.com/5656702/102898284-319fba00-4461-11eb-9586-3bd5bd87bbbb.png">

### How to test the changes in this Pull Request:
1. View [this branch's README](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/add/github-status-badges)
The badges are currently configured to show the status of actions run against the `trunk` branch. For now I think it will just suffice to see the badges appear in the right place in the README.
